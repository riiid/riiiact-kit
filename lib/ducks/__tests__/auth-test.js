import reducer, {
  requestAuth, receiveAuthSuccess, receiveAuthFailure
} from 'ducks/auth';

describe('reducers/auth', () => {
  it('should handle requestAuth()', () => {
    expect(reducer(undefined, requestAuth())).toMatchSnapshot();
  });

  it('should receiveAuthSuccess()', () => {
    expect(reducer(undefined, receiveAuthSuccess({}))).toMatchSnapshot();
  });

  it('should receiveAuthFailure()', () => {
    expect(reducer(undefined, receiveAuthFailure({}))).toMatchSnapshot();
  });
});
