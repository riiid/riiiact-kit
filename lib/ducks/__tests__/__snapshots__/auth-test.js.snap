// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers/auth should handle requestAuth() 1`] = `
Object {
  "authenticated": false,
  "user": null,
}
`;

exports[`reducers/auth should receiveAuthFailure() 1`] = `
Object {
  "authenticated": false,
  "error": Object {},
  "user": null,
}
`;

exports[`reducers/auth should receiveAuthSuccess() 1`] = `
Object {
  "authenticated": true,
  "error": null,
  "user": Object {},
}
`;
